wiki/Beatmap_Discussion/en.md

```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 ### Functions
 
 - `Collapse all`/`Expand all` and the up arrow: These will show/hide posts.
-- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on suggestions or problems.**
+- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on Suggestions or Problems.**
 - Timeline: Shows the timestamp the issue addresses. Only available in the Timeline section.
 - CREATOR icon: Displays responses/submissions from the user who made the beatmapset.
 - permalink, edit, delete: Permalink grabs a direct link to the post. Edit allows the submitter to edit their submission. Delete allows the submitter to remove their submission.
 - Respond: Reply to the submission. Pressing Enter or clicking Reply afterwards will submit your response.
 - Mark as Resolved: The creator may select this when the issue has been addressed and either accepted or denied.
 - Reopen a Discussion: If the submission needs further discussion, the creator or mod submitter may reopen the submission by responding and unchecking Resolved.
 
-## Earning Kudosu!
+## Hype, Nomination, and Kudosu!
+
+### Hype
 
-*Note: You get ten votes every hour.*
+Hype is used to help push beatmaps forward to nomination; similar to how Star Priority was previously used. Once a beatmapset receives 5 Hype, it becomes eligible for Nomination. Please note the following:
+
+- Each user has a limited supply of hype (5 per month to start).
+  - You will regain a hype 30 days after that one hype was used.
```

Comment:
after one hype was used

Reply:
I specifically choose to word it as `after that one` because it refers to the specific hype that was used at that time

will reword it to `You will regain a hype 30 days after it was used.`